core/pkg/server/config.go

```diff
@@ -103,6 +111,26 @@ func (runConfig *RunConfig) AddTelemetryAndMetrics(
 	}
 }
 
+// Serializes the run configuration to send to the backend.
+func (runConfig *RunConfig) Serialize(format ConfigFormat) ([]byte, error) {
```

Comment:
should we add this header: https://github.com/wandb/wandb/blob/main/wandb/sdk/lib/config_util.py#L48

Reply:
probably need to do it only when we call it from the file serialization callsite

Reply:
(i'm ok to skip as well, i think mid-term we shouldn't send this file at all as we already send all the config to infra anyways)

Reply:
That should probably happen in `Sender::writeAndSendConfigFile()` instead, since it's specifically for persisting to a file rather than sending to the backend. But I think we can skip it---if we want version-specific logic later, we can assume a missing version is the same as wandb_version=1.

Reply:
yep that's what i meant by other call-site, agreed. I think we mainly kept this file around for infra as a backup. So ideally we completely get away from it...